### PR TITLE
New package: RCharan.wiki version 1.0

### PR DESCRIPTION
**Pull request by charanravi-online**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/153706)

### CODE REVIEW

**Comment by charanravi-online** [comment]
@microsoft-github-policy-service agree

File: manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RCharan.wiki
+PackageVersion: "1.0"
+PackageLocale: en-US
+Publisher: R Charan
+PackageName: wiki
+License: MIT License
+ShortDescription: wiki Setup
```
Comment (stephengillie) [inline manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml:10]:
```suggestion
ShortDescription: A command-line tool to get Wikipedia summaries in your terminal. 
```

File: manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RCharan.wiki
+PackageVersion: "1.0"
+PackageLocale: en-US
+Publisher: R Charan
+PackageName: wiki
+License: MIT License
```
Comment (stephengillie) [inline manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml:9]:
```suggestion
License: Apache-2.0
```
I'm not a lawyer, just copying from the project page. If this shouldn't change, then please feel free to ignore the suggestion.

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**